sympy/solvers/solveset.py

```diff
@@ -1139,9 +1137,7 @@ def linsolve(system, *symbols):
     Examples
     ========
 
-    >>> from sympy.solvers.solveset import linsolve
-    >>> from sympy import Matrix, S
-    >>> from sympy import symbols
+    >>> from sympy import Matrix, S, linsolve, symbols
```

Comment:
It's used here https://github.com/sympy/sympy/pull/10049/files#diff-eec0422923e8f100745c015cd8fdd6cfR1170